aries_cloudagent/anoncreds/default/legacy_indy/registry.py

```diff
@@ -198,7 +198,7 @@ async def register_schema(
             if not profile.settings.get_value("wallet.type"):
                 # TODO is this warning necessary?
                 reason += ": missing wallet-type?"
-            raise AnonCredsRegistrationError(reason)
+            raise AnonCredsResolutionError(reason)
```

Comment:
It seems like this really ought to be a registration error rather than a resolution error; the operation attempted is a `register` rather than a `resolve` so, even in the event we were attempting to resolve something in order to register something immediately after, I think it is better for this to be a registration error from an interface perspective.

Reply:
updated related code